### DISCUSSION TITLE
Hide Reblogs from certain blogs only

### DISCUSSION THREAD

**Discussion by reallyreallybroly** [AprilSylph/XKit-Rewritten#1039] [Suggestion Box]
Hello
Thank you for working on Xkit.

I apologize if somebody else made this suggestion before. I've tried searching it but nothing came up.

I follow over 3000 blogs and a few of them only for their original content, for example artists that don't have art-only blogs. I love their art so I have to follow them but I'm not interested in having my dash filled with their reblogs.

I tried the Show Originals feature but as you'd expect, trying to add the names of all the blogs I follow for what they reblog will take too much effort.

I was wondering if you could add an option that turns off reblogs from certain blogs only, blogs I'd enter the names of in a box like in the Show Originals feature.

**Comment by marcustyphoon** [comment]
The not-currently-implemented `Mute` script sounds like just what you want! Feel free to follow issue #674.

**Reply by reallyreallybroly** [reply]
ooo it really does! thank you so much :3
